test/units/plugins/lookup/test_password.py

```diff
@@ -20,117 +20,349 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
+from units.mock.loader import DictDataLoader
 from ansible.compat.tests import unittest
+from ansible.compat.tests.mock import patch
 
-from ansible.plugins.lookup.password import LookupModule, _parse_parameters, DEFAULT_LENGTH
+from ansible.compat.six import text_type
+
+from ansible.errors import AnsibleError
+from ansible.plugins import PluginLoader
+from ansible.plugins.lookup import password
```

Comment:
Style-wise, I like to make the thing being tested the last import and visually separate from the other imports (a blank line between them and the import for the thing(s) being tested.